lib/sorcery/providers/facebook.rb

```diff
@@ -12,16 +12,17 @@ class Facebook < Base
 
       attr_reader   :mode, :param_name, :parse
       attr_accessor :access_permissions, :display, :scope, :token_url,
-                    :user_info_path
+                    :user_info_path, :authorized_url, :api_version
 
       def initialize
         super
 
         @site           = 'https://graph.facebook.com'
-        @user_info_path = '/me'
+        @user_info_path = 'me'
         @scope          = 'email,offline_access'
         @display        = 'page'
         @token_url      = 'oauth/access_token'
+        @authorized_url = 'oauth/authorize'
```

Comment:
Could you change name of this var to `auth_path`? This is the name we use with other providers (Github, Heroku)